### DISCUSSION TITLE
Add query parameters on filter

### DISCUSSION THREAD

**Discussion by juliardi** [orchidsoftware/platform#2819] [Q&A]
Hello, I have a tree-like data like this :
- folderA
  - childA1
    - childA1X
    - childA1Y
    - childA1Z
  - childA2
  - childA3
- folderB
  - childB1
  - childB2
    - childB2X
    - childB2Y
    - childB2Z
  - childB3
  
These tree data is loaded on a list screen based on the value of query parameter 'prefix'.
For example, if prefix=folderA, the list screen will only display :
- childA1
- childA2
- childA3

And if prefix=folderB/childB2, it will display :
- childB2X
- childB2Y
- childB2Z

etc.

Now, if I use the filter feature while prefix is not empty, it will load a new URL with `filter` parameter without including the `prefix` parameter. How do I include the `prefix` parameter when filtering?

**Comment by macit-emre** [comment]
+1
blabla->filter() needs a param to include other URL params.
